server/pages/appointments/enforcementPage.test.ts

```diff
@@ -39,24 +43,36 @@ describe('EnforcementPage', () => {
         return offender
       })
 
-      const result = page.viewData(appointment)
+      const result = page.viewData(appointment, form)
 
       expect(result.offender).toBe(offender)
     })
 
-    it('should return an object containing a back link ', async () => {
+    it('should return an object containing a back link to compliance if attended', async () => {
+      const contactOutcome = contactOutcomeFactory.build({ attended: true })
+      const attendedForm = appointmentOutcomeFormFactory.build({ contactOutcome })
       jest.spyOn(paths.appointments, 'logCompliance')
 
-      const result = page.viewData(appointment)
+      const result = page.viewData(appointment, attendedForm)
       expect(paths.appointments.logCompliance).toHaveBeenCalledWith({ appointmentId: appointment.id.toString() })
       expect(result.backLink).toBe(pathWithQuery)
     })
 
-    it('should return an object containing an update link for the form', async () => {
-      jest.spyOn(paths.appointments, 'confirm')
+    it('should return an object containing a back link to log hours if not attended', async () => {
+      const contactOutcome = contactOutcomeFactory.build({ attended: true })
+      const notAttendedForm = appointmentOutcomeFormFactory.build({ contactOutcome })
+      jest.spyOn(paths.appointments, 'logHours')
 
-      const result = page.viewData(appointment)
+      const result = page.viewData(appointment, notAttendedForm)
       expect(paths.appointments.logCompliance).toHaveBeenCalledWith({ appointmentId: appointment.id.toString() })
+      expect(result.backLink).toBe(pathWithQuery)
```

Comment:
As the path util is mocked we don't seem to be testing the logic of the `backPath` method. It doesn't need to be fixed now as it's beyond the point of this PR, but could be worth removing the mock to better test.